src/flashblocks/flashblocks-websocket-proxy/launcher.star

```diff
@@ -0,0 +1,79 @@
+_observability = import_module("/src/observability/observability.star")
+_ethereum_package_constants = import_module(
+    "github.com/ethpandaops/ethereum-package/src/package_io/constants.star"
+)
+
+_net = import_module("/src/util/net.star")
+
+
+def launch(
+    plan,
+    params,
+    conductors_contexts,
+    observability_helper,
+):
+    config = get_service_config(
+        plan=plan,
+        params=params,
+        conductors_contexts=conductors_contexts,
+        observability_helper=observability_helper,
+    )
+
+    service = plan.add_service(params.service_name, config)
+
+    ws_port = params.ports[_net.WS_PORT_NAME]
+    ws_url = "ws://{}:{}".format(service.name, ws_port.number)
```

Comment:
`_net.service_url` handles URL construction - you just need to set the `application_protocol` to `ws` on the `Port` / `PortSpec` object